colorPairCodeHelper.cpp

```diff
@@ -0,0 +1,19 @@
+#include "colorPairCode.h"
```

Comment:
test code implementation is in a different file. But the declaration of the test code functions are in file containing production code.

This is an improvement area where the test code is kept in a separate file and production code in separate file.
It helps in maintainability and also makes it easy to exclude the test code from shipping to field.